en/views/helpers/form.rst

```diff
@@ -16,39 +16,53 @@ Starting a Form
 
 .. php:method:: create(mixed $model = null, array $options = [])
 
+* ``$model`` - The context for which the form is being defined. Can be an ORM
+  entity, ORM resultset, array of metadata or ``false/null`` (to make a
+  model-less form).
+* ``$options`` - An array of options and/or HTML attributes.
+
 The first method you'll need to use in order to take advantage of the FormHelper
 is ``create()``. This method outputs an opening form tag.
 
 All parameters are optional. If ``create()`` is called with no parameters
 supplied, it assumes you are building a form that submits to the current
 controller, via the current URL. The default method for form submission is POST.
-If you were to call ``create()`` inside the view for UsersController::add(), you
-would see something like the following output in the rendered view:
+If you were to call ``create()`` inside the view for ``UsersController::add()``,
+you would see something like the following output in the rendered view:
 
 .. code-block:: html
 
     <form method="post" action="/users/add">
 
 The ``$model`` argument is used as the form's 'context'. There are several
-built-in form contexts and you can add your own, which we'll cover in the next
-section. The built-in providers map to the following values of ``$model``:
-
-* An ``Entity`` instance or, an iterator map to the ``EntityContext``, this
-  context allows FormHelper to work with results from the built-in ORM.
-* An array containing the ``schema`` key, maps to ``ArrayContext`` which allows
-  you to create simple data structures to build forms against.
-* ``null`` and ``false`` map to the ``NullContext``, this context class simply
-  satisifies the interface FormHelper requires. This context is useful if you
-  want to build a short form that doesn't require ORM persistence.
-
-All contexts classes also have access to the request data, making it simpler to
+built-in form contexts and you can add your own, which we'll cover below, in
+a following section. The built-in providers map to the following values of
+``$model``:
+
+* An ``Entity`` instance or an iterator will map to
+  `EntityContext <https://api.cakephp.org/3.3/class-Cake.View.Form.EntityContext.html>`_;
```

Comment:
Given that the contexts don't have an book docs (as its not something the lay person has to interact with direclty) perhaps an API link is reasonable.